projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader.component.ts

```diff
@@ -24,25 +26,25 @@ export class NgxSkeletonLoaderComponent implements OnInit, AfterViewInit, OnDest
   static ngAcceptInputType_animation: boolean | string;
 
   @Input()
-  count = 1;
+  count: NgxSkeletonLoaderConfig['count'] = this.configService.config.count;
 
   @Input()
-  loadingText = 'Loading...';
+  loadingText: NgxSkeletonLoaderConfig['loadingText'] = this.configService.config.loadingText;
 
   @Input()
-  appearance: 'circle' | '' = '';
+  appearance: NgxSkeletonLoaderConfig['appearance'] = this.configService.config.appearance;
 
   @Input()
-  animation: 'progress' | 'progress-dark' | 'pulse' | 'false' | false = 'progress';
+  animation: NgxSkeletonLoaderConfig['animation'] = this.configService.config.animation;
 
-  // This is required since ngStyle is using `any` as well
-  // More details in https://angular.io/api/common/NgStyle
-  // tslint:disable-next-line: no-any
-  @Input() theme: { [k: string]: any } = {};
+  @Input()
+  theme: NgxSkeletonLoaderConfig['theme'] = this.configService.config.theme;
```

Comment:
This one is kinda tricky, so it requires a type. Otherwise, `npm run build:ssr` will fail because the code is complaining about the theme type used on the template .

What can be done in this case is:

#1 - Change the types to expose `NgxSkeletonLoaderConfigTheme` with the theme configuration
#2 - Change this line to be like that
```suggestion
  theme: NgxSkeletonLoaderConfig['theme'] = this.configService.config.theme ?? null;
```